spring-integration-mail/src/test/java/org/springframework/integration/mail/TestMailServer.java

```diff
@@ -282,10 +282,10 @@ else if (line.contains("FETCH 1 (ENVELOPE")) {
 							write("* 1 FETCH (RFC822.SIZE 6909 INTERNALDATE \"27-May-2013 09:45:41 +0000\" "
 									+ "FLAGS (\\Seen) "
 									+ "ENVELOPE (\"Mon, 27 May 2013 15:14:49 +0530\" "
-									+ "\"Test Email\" ((\"Foo\" NIL \"foo\" \"bar.tv\")) "
+									+ "\"Test Email\" ((\"Bar\" NIL \"bar\" \"baz\")) "
 									+ "((\"Foo\" NIL \"foo\" \"bar.tv\")) "
 									+ "((\"Foo\" NIL \"foo\" \"bar.tv\")) "
-									+ "((\"Bar\" NIL \"bar\" \"baz.net\")) NIL NIL "
+									+ "((\"Foo\" NIL \"foo\" \"bar\")) NIL NIL "
```

Comment:
I assumed you were just complaining that the from/to were not consistent when using a mapper, so I just fixed up the envelope and MESSAGE so they had consistent from/to.

The test server was not really intended for testing mail contents, just the mail protocols and proper operation of our adapters, but it makes sense to produce consistent results.